connectors/runner.py

```diff
@@ -16,6 +16,13 @@
 import time
 import functools
 
+# activate uvloop if present
+try:
+    import uvloop
```

Comment:
If we decided to use `uvloop`, we don't need this check? Simply use it directly?

```python
import uvloop

asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
```

Reply:
I pushed the patch for Crest to try it out on a bench; if the results are good, we will need to make the change more cleanly because `uvloop` is not supported on `win32`

So we'd move this code in the runner and provide a new option, e.g., `--event-loop uvloop`

From my recent profiling, we spend a lot of time calling `kqueue.select` and it's probable that switching to `uvloop` will improve performances. `libuv` is the loop used by NodeJS and it usually makes Py app like our faster than Go apps :)